src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="category-edit">
+    <div v-if="doneMessage" class="category-edit__notice--update">
+      <app-text bg-success>{{ doneMessage }}</app-text>
+    </div>
+    <div class="category-edit__columns">
+      <section class="category-edit-editor">
+        <app-heading :level="1">カテゴリー管理</app-heading>
+        <app-router-link
+          block
+          underline
+          key-color
+          hover-opacity
+          to="/categories"
+          class="category-edit-editor-back"
+        >
+          カテゴリー一覧へ戻る
+        </app-router-link>
+        <div class="category-edit-form">
+          <app-input
+            v-validate="'required'"
+            name="name"
+            type="text"
+            placeholder="カテゴリー名を入力してください。"
+            white-bg
+            data-vv-as="カテゴリー名"
+            :error-messages="errors.collect('name')"
+            :value="categoryName"
+            @update-value="$emit('category-name', $event)"
+          />
+        </div>
+        <app-button
+          class="category-edit-submit"
+          button-type="submit"
+          round
+          :disabled="!disabled"
```

Comment:
[imo]

`!disabled`（disabled が false の時、つまり無効じゃない）の時に `:disabled` props が true になるのに違和感を感じます。
以下のような記述にできてかつ動作するとわかりやすくなるかなと思うので修正をお願いします。

```suggestion
          :disabled="disabled"
```

Reply:
修正しました。

Reply:
@nodoka-kobayashi 
[must]
ここは`null`ではなく`:disabled="disabled"`にしましょう！
`null`だと更新中にボタンが非活性にならなくなってしまいます！

Reply:
:disabled="disabled"に修正しました。